extensions/flags/js/src/forum/components/FlagList.tsx

```diff
@@ -16,49 +18,53 @@ export interface IFlagListAttrs extends ComponentAttrs {
 export default class FlagList<CustomAttrs extends IFlagListAttrs = IFlagListAttrs> extends Component<CustomAttrs, FlagListState> {
   oninit(vnode: Mithril.Vnode<CustomAttrs, this>) {
     super.oninit(vnode);
-    this.state = this.attrs.state;
   }
 
   view() {
-    const flags = this.state.cache || [];
+    const state = this.attrs.state;
 
     return (
       <HeaderList
         className="FlagList"
         title={app.translator.trans('flarum-flags.forum.flagged_posts.title')}
-        hasItems={flags.length}
-        loading={this.state.loading}
+        hasItems={state.hasItems()}
+        loading={state.isLoading()}
         emptyText={app.translator.trans('flarum-flags.forum.flagged_posts.empty_text')}
+        loadMore={() => state.hasNext() && !state.isLoadingNext() && state.loadNext()}
       >
-        <ul className="HeaderListGroup-content">
```

Comment:
any reason why the `ul` with `li` items is gone now?

Reply:
That was an accident! I will add it back.